src/config/entities/__tests__/configuration.ts

```diff
@@ -183,7 +183,7 @@ export default (): ReturnType<typeof configuration> => ({
     relay: true,
     swapsDecoding: true,
     historyDebugLogs: false,
-    auth: false,
+    auth: true,
```

Comment:
We aligned that this was supposed to be opt in order to prevent changes across all the tests. Also changes to those tests seem out of scope for the Email Retrieval.

Reply:
This was a leftover from issues I encountered with tests. I've changed it back to `false` and only opted into `auth` where necessary. It's reduced the footprint of configuration overrides as well.